services/grafana-loki/0.69.10/grafana-loki-helmrelease/grafana-loki.yaml

```diff
@@ -1,46 +0,0 @@
-apiVersion: helm.toolkit.fluxcd.io/v2beta1
-kind: HelmRelease
-metadata:
-  name: grafana-loki
-  namespace: ${releaseNamespace}
-spec:
-  chart:
-    spec:
-      chart: loki-distributed
-      sourceRef:
-        kind: HelmRepository
-        name: grafana.github.io
-        namespace: kommander-flux
-      version: 0.69.10
-  interval: 15s
-  install:
-    crds: CreateReplace
-    remediation:
-      retries: 30
-  upgrade:
-    crds: CreateReplace
-    remediation:
-      retries: 30
-  releaseName: grafana-loki
-  valuesFrom:
-    - kind: ConfigMap
-      name: grafana-loki-0.69.10-d2iq-defaults
-  # upstream chart doesn't apply loki.annotations to all loki deployment/statefulsets
-  # TODO: remove this after https://github.com/grafana/helm-charts/issues/1905 is fixed
-  postRenderers:
```

Comment:
btw @cwyl02 looks like your PR was merged if you wanna verify and close out your upstream issue https://github.com/grafana/helm-charts/issues/1905 !